src/python/ensembl/io/genomio/gff3/simplifier.py

```diff
@@ -484,14 +484,15 @@ def normalize_mirna(self, gene: SeqFeature) -> List[SeqFeature]:
 
         # Passed the checks
         primary = transcripts[0]
-
         logging.debug(f"Formatting miRNA gene {gene.id}")
 
         new_genes = []
         new_primary_subfeatures = []
         num = 1
         for sub in primary.sub_features:
             if sub.type == "exon":
+                gene.type = "ncRNA_gene"
+                primary.type = "miRNA_primary_transcript"
                 new_primary_subfeatures.append(sub)
```

Comment:
I would put the biotype change at the earlier stage where we create the `primary` and the `gene`  (472-473)

Reply:
oh yes makes sense ! I have forgotten to make that change.